src/components/ListItem.js

```diff
@@ -1,16 +1,16 @@
 import React, { useState, useEffect } from 'react';
 import { db } from '../lib/firebase.js';
 import { calculateEstimate } from '@the-collab-lab/shopping-list-utils';
-import { NavLink } from 'react-router-dom';
 import { collection, doc, onSnapshot, updateDoc } from 'firebase/firestore';
 import Footer from './Footer';
 import { useHistory } from 'react-router-dom';
 import { searchListHandler } from './customFilter.js';
+import { SortItems } from './Sort.js';
 
 const ListItem = () => {
   const [items, setItems] = useState([]);
   const [loading, setLoading] = useState(false);
-  const [error, setError] = useState(false);
+  const [error] = useState(false);
```

Comment:
Here too. it's more efficient to use a variable if the value won't change over time.